pallets/gear/src/runtime_api.rs

```diff
@@ -305,7 +286,7 @@ where
             min_limit,
             reserved,
             burned,
-            may_be_returned,
+            may_be_returned: 0,
```

Comment:
> @breathx, consider remove the field

To be used in future. Also removal will trigger api changes in js, so it will contain several versions of exactly the same api